utils/ExampleRunner/template-config.js

```diff
@@ -105,6 +105,12 @@ module.exports = {
     open: false,
     port: ${process.env.CS3D_PORT || 3000},
     historyApiFallback: true,
+    allowedHosts: [
```

Comment:
And need this example to test access on different hostnames.  I can omit this, but it took me a while to find the example configuration that worked.